WP Support Forum: Update RELEASE-NOTES.txt

ravishanker opened the pull request:
Update release notes for Community forum option on Help and Support screen.


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
NA

3. What automated tests I added (or what prevented me from doing so)
NA

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

ravishanker committed:
Update RELEASE-NOTES.txt

Update release notes for Community forum option on Help and Support screen.

irfano commented:
Thanks for handling this.  👍🏻
> [internal][WordPress-only] Adds new Community forums option on Help and Support Screen.

Should we also mention removing "Contact Support"?

ravishanker committed:
Update RELEASE-NOTES.txt

Update release notes text after review

ravishanker commented:
> Should we also mention removing "Contact Support"?

Actually, added that first then thought it would become long sentence.  Updated it to read better now.  Wdyt?

> Help & Support: replaces Contact Support with new Community forums, and removes Browse our FAQ.

irfano reviewed (approved):
Thank you! I just wanted to be sure you considered all changes. LGTM! 👍🏻
